bears/lua/LuaLintBear.py

```diff
@@ -18,6 +18,7 @@ class LuaLintBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Unreachable Code', 'Unused Code'}
```

Comment:
naaah that can't be this linter will be able to do more stuff! Looking at the docs we have e.g. variable misuse: `accessing uninitialized variable a`